### ISSUE TITLE
Update to 0.1.4 issue

### ISSUE THREAD

**Issue by mdilger** [cdjackson/HABmin2#157]
Hi,

I have tried to update to the latest version of HABmin2 after stopping OH2 and deleting the 0.1.3 file.

After restarting OH2 and then going the HABmin it is still reporting that it is v0.1.3 on the home page from HABmin.  

Have I done something wrong with my update process or is it just a typo on the home page?

Thanks,
Mike

**Comment by cdjackson** [comment]
What date is it reporting?  I'm not sure if I checked the version was correct before I released it (although it's auto generated, so it should be!) - I do know that the date/time were displayed correctly on the front page though.

If the time doesn't say 2245 on the 10th (approximaetly) then you're still running the old version, and it might be that Karaf has cached the old version. IF so, log on to the Karaf console, do a 'list' command to see what's running...
